remote-access/ip-address.md

```diff
@@ -65,7 +65,7 @@ nmap -sn 192.168.1.0/24
 Ping scan just pings all the IP addresses to see if they respond. For each device that responds to the ping, the output shows the hostname and IP address like so:
 
 ```
-Starting Nmap 6.40 ( http://nmap.org ) at 2014-03-10 12:46 GMT
+Starting Nmap 6.40 ( https://nmap.org/ ) at 2014-03-10 12:46 GMT
```

Comment:
The link here is actually in quoted log output, so should probably remain as http ?

Reply:
Yes

Reply:
Or replace the line with this?:
```
Starting Nmap 7.40 ( https://nmap.org ) at 2020-05-29 00:25 CEST
```
(copied from my Raspberry Pi as-is)